lib/token-assert.js

```diff
@@ -101,6 +101,8 @@ TokenAssert.prototype.sameLine = function(options) {
  * @param {Object} options.token
  * @param {Object} options.nextToken
  * @param {Object} [options.message]
+ * @param {Number} [options.lines=1] Amount of lines between tokens.
```

Comment:
It's not a real amount of lines between. It's more like difference between line numbers. Not sure how to name that properly. Gap?